### PR TITLE
Add response-policy override option to DNS support-server

### PR DESCRIPTION
**Pull request by kravciak**
Local run: I used `updates.suse.com A 68.232.34.1` to check that change was applied
 - http://dhcp91.suse.cz/tests/1682#step/first_boot/19

### CODE REVIEW

File: data/supportserver/named/db.rpz [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$TTL 60
+@            IN    SOA  localhost. root.localhost.  (
+                      2   ; serial 
+                      3H  ; refresh 
+                      1H  ; retry 
+                      1W  ; expiry 
+                      1H) ; minimum 
+              IN    NS    localhost.
+
+updates.suse.com        A    11.22.33.44
```
Comment (drpaneas) [inline data/supportserver/named/db.rpz]: Can we use the variable EDGECAST instead of the `11.22.33.44`

File: tests/caasp/first_boot.pm [hunk 2]
```diff
@@ -59,6 +59,8 @@ sub run {
         }
         # End of workaround
     }
+
+    record_info "RZP ping should fail" if script_run "ping -c1 updates.suse.com";
```
Comment (drpaneas) [inline tests/caasp/first_boot.pm]: what exactly are you testing with that? First boot it *after* the installation.

File: data/supportserver/named/db.rpz (hunk 1 above)
Comment (kravciak) [reply data/supportserver/named/db.rpz]: sure, that's just for testing that it really changes things

File: tests/caasp/first_boot.pm (hunk 2 above)
Comment (kravciak) [reply tests/caasp/first_boot.pm]: same as before, just to check that I can't ping 11.22.33.44

**Comment by drpaneas** [comment]
I am testing it: http://skyrim.qam.suse.de/tests/overview?version=2.0&build=4537-kravciak&distri=caasp&groupid=98

**Comment by coolo** [comment]
please revert all of this now - the DNS setup in SUSE was fixed

**Comment by kravciak** [comment]
PR here - https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4688

**Pull request merged**